.github/workflows/pr-reviewed.yml

```diff
@@ -57,6 +57,21 @@ jobs:
                       issue_number: pull.number,
                       name: "triage"
                     });
+                    try {
```

Comment:
Out of curiosity, why do we have extra logging specifically for removing the needs-attention label? We don't seem to have similar logging for other labels like triage.

Reply:
Since the issue search is filtering for `label:triage`, we know all the found PRs will have that label. We do log something for each of this (on L53)

```
console.log("Removing 'triage' label from PR " + pull.number + " : " + pull.title);
```

To be consistent, I thought we should log a message when we remove "needs-attention".

---

This log line is a bit extraneous. I'll remove it

```
console.log("PR did not have 'needs-attention' label");
```

Reply:
To ensure both "removeLabel" are executed, should we implement error handling for removing "triage"?

Reply:
Hm, this actually makes me realize that `triage` could be missing. I've modified the workflow to search for both labels separately.